[WARN] warning CS0162: Unreachable code detected

Noticed H+ throws this warning when loaded.
Packages\com.dreadscripts.hierarchyplus\Editor\SavedSettings.cs(67,10): warning CS0162: Unreachable code detected

--- a/Editor/SavedSettings.cs
+++ b/Editor/SavedSettings.cs
@@ -18,7 +18,6 @@ namespace DreadScripts.HierarchyPlus
 		internal static SavedSettings settings => data;
 
 		private const string PrefsKey = "HierarchyPlusSettingsJSON";
-		private const bool isEditor = true;
 
 		#region Main
 
@@ -63,8 +62,7 @@ namespace DreadScripts.HierarchyPlus
 
 				string rawData = dataBuilder.ToString();
 				string compressedData = CompressString(rawData);
-				if (isEditor) EditorPrefs.SetString(PrefsKey, compressedData);
-				else PlayerPrefs.SetString(PrefsKey, compressedData);
+				PlayerPrefs.SetString(PrefsKey, compressedData);
 			}
 		}
 
@@ -74,10 +72,7 @@ namespace DreadScripts.HierarchyPlus
 			{
 				string fullData = string.Empty;
 
-				if (isEditor && EditorPrefs.HasKey(PrefsKey))
-					fullData = EditorPrefs.GetString(PrefsKey, string.Empty);
-				else if (!isEditor && PlayerPrefs.HasKey(PrefsKey))
-					fullData = PlayerPrefs.GetString(PrefsKey, string.Empty);
+				fullData = PlayerPrefs.GetString(PrefsKey, string.Empty);
 				if (!string.IsNullOrWhiteSpace(fullData))
 					fullData = DecompressString(fullData);
 

--- a/Editor/SavedSettings.cs
+++ b/Editor/SavedSettings.cs
@@ -62,7 +62,7 @@ namespace DreadScripts.HierarchyPlus
 
 				string rawData = dataBuilder.ToString();
 				string compressedData = CompressString(rawData);
-				PlayerPrefs.SetString(PrefsKey, compressedData);
+				EditorPrefs.SetString(PrefsKey, compressedData);
 			}
 		}
 
@@ -71,8 +71,7 @@ namespace DreadScripts.HierarchyPlus
 			try
 			{
 				string fullData = string.Empty;
-
-				fullData = PlayerPrefs.GetString(PrefsKey, string.Empty);
+				fullData = EditorPrefs.GetString(PrefsKey, string.Empty);
 				if (!string.IsNullOrWhiteSpace(fullData))
 					fullData = DecompressString(fullData);
 

